bundle-size/index.js

```diff
@@ -0,0 +1,92 @@
+// @ts-check
+
+const {
+  getComment,
+  prMessageSymbol,
+  getBelowThresholdComment,
+} = require("./comment");
+const { compareStats } = require("./compareStats");
+
+module.exports = async (
+  { core, context, github },
+  threshold,
+  mainStatsFile,
+  prStatsFile
+) => {
+  try {
+    const {
+      payload: { pull_request },
+      repo,
+    } = context;
+    const prNumber = pull_request.number;
+    console.log("Comparing stats... 🔍");
+    const { assetsDiff, modulesDiff, entriesDiff } = compareStats(
+      mainStatsFile,
+      prStatsFile
+    );
+    console.log("Comparing stats done. 👍");
+    const diffThreshold = parseInt(threshold, 10);
+    const commentBody = getComment(assetsDiff, modulesDiff, entriesDiff);
+    console.log("Checking PR comments... 📝");
+    const { data: comments } = await github.rest.issues.listComments({
+      ...repo,
+      issue_number: prNumber,
+    });
+
+    const [previousComment, ...restComments] = comments.filter(
+      (comment) => comment.body && comment.body.includes(prMessageSymbol)
+    );
+
+    if (restComments.length > 1) {
+      console.log("Cleaning up old comments... 🧹");
```

Comment:
are you certain we are removing "old" comments and not keeping the oldest and removing the newer ones?

Doesn't really matter for this case, but making sure there's no important reasoning behind updating only the latest